apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -705,15 +729,16 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
                                                             ".inputs.validations.empty", { fieldName })
                                                     }
                                                     type="dropdown"
-                                                    children={ countryList ? countryList.map(list => {
-                                                        return {
-                                                            "data-testid": `${testId}-` + list.value as string,
-                                                            flag: list.flag,
-                                                            key: list.key as string,
-                                                            text: list.text as string,
-                                                            value: list.value as string
-                                                        };
-                                                    }) : [] }
+                                                    children={ countryList
+                                                        ? countryList.map((list: DropdownItemProps) => {
+                                                            return {
+                                                                "data-testid": `${testId}-` + list.value as string,
```

Comment:
We can use the template literal instead of concatenation, right?

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/3763/commits/25d7975004f07604b9d6adb6dd1e18041b8ef502